_irc/index.md

```diff
@@ -6,15 +6,21 @@ meta-description: Specifications that the IRCv3 Working Group maintains and dist
 index: true
 ---
 
-Right now the IRCv3 specification is distributed as a series of extension
-specifications to IRC protocol version 2.7, also known as
-[RFC1459](http://tools.ietf.org/html/rfc1459). To understand the basis of the
-IRC version 3 protocol, please read RFC1459 followed by the extension
-specifications.
-
-IRCv3 is based on a set of specifications. Initially these specifications were
-released as a versioned distribution – these days, however, each specification
-is released when it is stable and has been widely tested.
+IRCv3 specifications build on top of the core IRC protocol. The primary core
+IRC protocol specifications are [RFC1459](http://tools.ietf.org/html/rfc1459)
+and [RFC7192](https://tools.ietf.org/html/rfc7194), although one of our members
+has been working on updated core protocol specifications
+[here](http://modern.ircdocs.horse/) which you may find useful to consult.
+If you have any questions on the core IRC protocol, please feel free to ask us
+[directly]({{site.baseurl}}/contact.html) or with an issue in our
+[feedback](https://github.com/ircv3/ircv3-feedback) Github repository.
+
+To fully understand IRCv3, please read the core IRC protocol specifications
+followed by the IRCv3 specifications.
+
+Initially these specifications were released as a versioned distribution, but
```

Comment:
That sentence is going to date quickly, maybe something like "up until version 3.2 (or a date?), specifications were released as a versioned distribution. This is no longer the case and each specification is now released when ..."

or "The specifications are released when they are stable and have been wildly testing. In the past, IRCv3 adopted a versioning scheme..."